packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/FixtureExtensions.kt

```diff
@@ -162,26 +161,12 @@ fun RemoteRobot.openProject(absolutePath: String) {
         """,
     )
 
-    maybeTerminateButton()
     maybeIdeaFrame()?.closeAllFiles()
-    maybeTerminateButton()
 }
 
 /**
  * Closes the project and waits until properly closed.
  */
-fun RemoteRobot.closeProject() {
+fun RemoteRobot.closeProject() = step("Closing any open project") {
     invokeAction("CloseProject")
-    maybeTerminateButton()
-}
-
-private fun RemoteRobot.maybeTerminateButton() {
```

Comment:
This was necessary when a process is running when a test finishes (like a gradle sync). It opens a pop up with the button. Why are we getting rid of this? Is not necessary anymore?